Bender.lock

```diff
@@ -235,3 +245,9 @@ packages:
     source:
       Git: https://github.com/pulp-platform/timer_unit.git
     dependencies: []
+  zeroriscy:
```

Comment:
Why are we adding a zeroriscy in the bender.lock?

Reply:
It is a dependency of Neureka